locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -3198,14 +3227,26 @@ msgstr "Un lien de réinitialisation a été envoyé à %s."
 #~ msgid "Reactivate user"
 #~ msgstr "Réactiver le compte"
 
-#~ msgid "replied to <a href=\"%(user_path)s\">%(username)s's</a> <a href=\"%(status_path)s\">review</a>"
-#~ msgstr "a répondu à la <a href=\"%(status_path)s\">critique</a> de <a href=\"%(user_path)s\">%(username)s</a>"
-
-#~ msgid "replied to <a href=\"%(user_path)s\">%(username)s's</a> <a href=\"%(status_path)s\">comment</a>"
-#~ msgstr "a répondu au <a href=\"%(status_path)s\">commentaire</a> de <a href=\"%(user_path)s\">%(username)s</a>"
-
-#~ msgid "replied to <a href=\"%(user_path)s\">%(username)s's</a> <a href=\"%(status_path)s\">quote</a>"
-#~ msgstr "a répondu à la <a href=\"%(status_path)s\">citation</a> de <a href=\"%(user_path)s\">%(username)s</a>"
+#~ msgid ""
+#~ "replied to <a href=\"%(user_path)s\">%(username)s's</a> <a href="
+#~ "\"%(status_path)s\">review</a>"
+#~ msgstr ""
+#~ "a répondu à la <a href=\"%(status_path)s\">critique</a> de <a href="
+#~ "\"%(user_path)s\">%(username)s</a>"
+
+#~ msgid ""
+#~ "replied to <a href=\"%(user_path)s\">%(username)s's</a> <a href="
+#~ "\"%(status_path)s\">comment</a>"
+#~ msgstr ""
+#~ "a répondu au <a href=\"%(status_path)s\">commentaire</a> de <a href="
+#~ "\"%(user_path)s\">%(username)s</a>"
+
+#~ msgid ""
+#~ "replied to <a href=\"%(user_path)s\">%(username)s's</a> <a href="
+#~ "\"%(status_path)s\">quote</a>"
+#~ msgstr ""
+#~ "a répondu à la <a href=\"%(status_path)s\">citation</a> de <a href="
+#~ "\"%(user_path)s\">%(username)s</a>"
```

Comment:
Not quite sure if these lines should be updated either, although they should not cause any trouble since they’re all commented.

Reply:
Ah yes, these could even be deleted, from what I gather it's old translation keys that don't exist anymore